crates/subspace-farmer/src/plotting.rs

```diff
@@ -154,7 +157,7 @@ async fn background_plotting<T: RpcClient + Clone + Send + 'static>(
             // TODO: Erase plot
         }
 
-        drop(plot);
+        //drop(plot);
```

Comment:
Why?

Reply:
Without this, it creates an error in the plotting continue test (if you are asking why keeping the comment, I thought it might be useful for the later maybe, but removed now).

Reply:
What error? It should not have any errors because of that.

Reply:
Inspecting. I might be confusing this with the other `drop(plot)`. Will give a detailed report soon